metrics-graphite/src/main/java/com/codahale/metrics/graphite/Graphite.java

```diff
@@ -124,6 +124,7 @@ public void connect() throws IllegalStateException, IOException {
         }
 
         if (address.getAddress() == null) {
+            address = null;
             throw new UnknownHostException(address.getHostName());
```

Comment:
This will cause a `NullPointerException` in the following line. 😉

Reply:
ho sorry